examples/prescriptors/neat/utils.py

```diff
@@ -64,9 +68,10 @@ def prepare_historical_df():
                   parse_dates=['Date'],
                   encoding="ISO-8859-1",
                   error_bad_lines=False)
```

Comment:
fill na with empty string to match raw data loading in `validation.scenario_generator`